scapy/layers/dhcp.py

```diff
@@ -88,6 +91,90 @@ def getfield(self, pkt, s):
             ret.append(val)
         return b"", [x[0] for x in ret]
 
+class ClasslessStaticRoutesField(FieldListField):
+    """
+    RFC 3442 defines classless static routes as up to 9 bytes per entry:
+     Code Len Destination 1    Router 1
+    +-----+---+----+-----+----+----+----+----+----+
+    | 121 | n | d1 | ... | dN | r1 | r2 | r3 | r4 |
+    +-----+---+----+-----+----+----+----+----+----+
+    Destination first byte contains one octet describing the width followed
+    by all the significant octets of the subnet.
+    """
+    def m2i(self, pkt, x):
+        # type: (Packet, bytes) -> str
+        # b'\x20\x01\x02\x03\x04\t\x08\x07\x06' -> (1.2.3.4/32:9.8.7.6)
+        offset = 0
+        if six.PY2:
+            prefix = struct.unpack('B', x[offset])[0]
```

Comment:
Could you use `prefix = orb(x[offset])` instead of this? works for all versions (`scapy.compat.orb`)

Reply:
done! did it for the other spot using `struct.unpack`

Reply:
No you can replace the entire if/else with that :)